app/server/app.js

```diff
@@ -9,17 +9,13 @@ const indexPath = path.join(__dirname, '../index.html');
 
 app.use(publicPath);
 
-app.get('/', (request, response) => {
-  response.sendFile(indexPath);
+app.get('/', (req, res) => {
+  res.sendFile(indexPath);
 });
 
-app.get('/articles', searchArticles, (request, response) => {
-  response.status(200);
-
-  // sends back an array of articles for a single source
-  // not an array of sources
-  const { articles } = request;
-  response.json(articles);
+app.get('/articles', searchArticles, (req, res) => {
+  const { articles } = req;
+  res.json(articles);
```

Comment:
Can we go back to request and response instead of req and res? I have a harder time quickly seeing the difference between req/res v. request/response and sometimes end up with syntax errors because of it. It's a bit more verbose, but more explicit and less prone to mistakes.